### PR TITLE
adding new info for ARM credentials and publication

### PR DESCRIPTION
**Pull request by emuus**
Also moving the info for "my account" and "credentials" just after "getting started" as this seems more high level and intro type documentation

### CODE REVIEW

File: end-user/en/source/pages/changelog.rst [hunk 1]
```diff
@@ -11,6 +11,9 @@ Release Date: 2017-09-04
 New Features
 ~~~~~~~~~~~~
 * Introduce a REST API for users to list files to synchronize to the target environment after a CentOS scan without overlay.
+* Updated information to be entered in ``Credentials`` for Azure ARM connector.
```
Comment (ludop-usharesoft) [inline end-user/en/source/pages/changelog.rst]: I'm not sure I have explain you the goal of these modifications sorry.
In fact before the image was just publish as a "vhd" blob file in the Azure cloud Account, but it was not accessible in the "image" menu of the cloud. Now there is a additional step in our connector that consist in creating an image accessible in the cloud console from this blob file.
It simplifies how the user can create a VM from this.

File: end-user/en/source/pages/credentials/user-cloud-accounts.rst [hunk 2]
```diff
@@ -7,13 +7,36 @@ Managing Cloud Accounts
 
 In order to publish an image to a cloud using UForge AppCenter, you will need to add your cloud credentials to UForge. You must have a cloud account prior to setting up your credentials on the platform. Have all the information for your cloud account on hand before starting.
 
+.. note:: If you are an Amazon IAM user you will need to generate an X.509 Certificate following `these instructions <http://www.dowdandassociates.com/blog/content/howto-generate-an-x-dot-509-certificate-for-an-amazon-iam-user/>`_ prior to creating your cloud account on UForge.
```
Comment (ludop-usharesoft) [inline end-user/en/source/pages/credentials/user-cloud-accounts.rst]: This step is mandatory only for the "azure classic" connector.

File: end-user/en/source/pages/credentials/user-cloud-accounts.rst [hunk 3]
```diff
@@ -7,13 +7,36 @@ Managing Cloud Accounts
...
+.. _azure-account:
+
+Setting up a Microsoft Azure Resource Manager Account
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to publish images to Microsoft ARM, you must first set up the correct Azure ARM credentials in UForge. To do so, first, from your Microsoft Azure account you will need to:
+
+	1. Create an Azure Active Directory application (as described in the `official Microsoft Azure documentation <https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application>`_).
+	2. Get the application ID and authentication key (as described in `official Microsoft Azure documentation <https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key>`_).
+	3. Retrive the tenant ID which you will need to enter in UForge credentials (as described in `official Microsoft Azure documentation <https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id>`_).
```
Comment (ludop-usharesoft) [inline end-user/en/source/pages/credentials/user-cloud-accounts.rst]: retrive => retrieve ?

File: end-user/en/source/pages/changelog.rst (hunk 1 above)
Comment (emuus) [inline end-user/en/source/pages/changelog.rst]: ok

File: end-user/en/source/pages/templating/appliance-publish.rst [hunk 4]
```diff
@@ -21,7 +21,17 @@ In order to publish a machine image to a cloud environment or container, you mus
 
 		.. image:: /images/machine-image-publish.png
 
-	4. Following the instructions, choose the account and fill in any additional information required.
+	4. Following the instructions in the tooltips, choose the account and fill in any additional information required.
+
+	.. note:: When publishing to Microsoft Azure Resource Manager you should note the following restrictions:
+
+		* The storage account must exist in your Microsoft account
+		* The container is not mandatory. It will be automatically created if left empty.
```
Comment (emuus) [inline end-user/en/source/pages/templating/appliance-publish.rst]: ok

File: end-user/en/source/pages/credentials/user-cloud-accounts.rst (hunk 2 above)
Comment (emuus) [inline end-user/en/source/pages/credentials/user-cloud-accounts.rst]: ok will add this specification

File: end-user/en/source/pages/credentials/user-cloud-accounts.rst (hunk 3 above)
Comment (emuus) [inline end-user/en/source/pages/credentials/user-cloud-accounts.rst]: yes

File: end-user/en/source/pages/templating/appliance-publish.rst [hunk 5]
```diff
@@ -26,7 +26,7 @@ In order to publish a machine image to a cloud environment or container, you mus
 	.. note:: When publishing to Microsoft Azure Resource Manager you should note the following restrictions:
 
 		* The storage account must exist in your Microsoft account
-		* The container is not mandatory. It will be automatically created if left empty.
+		* The container is mandatory. If the value your set does not match an existing container it will be created with the new name you have set.
```
Comment (ludop-usharesoft) [inline end-user/en/source/pages/templating/appliance-publish.rst]: Small typo with "your" instead of "you"

Comment (emuus) [inline end-user/en/source/pages/templating/appliance-publish.rst]: already fixed

**Comment by emuus** [comment]
I have added the new step and screen for subscription ID

File: admin/en/source/pages/changelog.rst [hunk 6]
```diff
@@ -11,6 +11,9 @@ Release Date: 2017-09-04
 New Features
 ~~~~~~~~~~~~
 * Introduce a REST API for users to list files to synchronize to the target environment after a CentOS scan without overlay.
+* Microsoft Azure connector has been updated. Previously with UForge the machine image was publish as a "vhd" blob file in the Azure cloud Account. Now an image will be accessible in the cloud console from this blob file. In order to support this additional information must be entered in ``Credentials`` for Azure ARM connector.
+
+	.. note:: If you have an existing Azure ARM account already setup in UForge, then you must update the credential information with the tenant id.
```
Comment (jbremond) [inline admin/en/source/pages/changelog.rst]: tenant id is not the only field the user need to fill. So we can say something more global I think

File: admin/en/source/pages/changelog.rst [hunk 7]
```diff
@@ -11,6 +11,9 @@ Release Date: 2017-09-04
 New Features
 ~~~~~~~~~~~~
 * Introduce a REST API for users to list files to synchronize to the target environment after a CentOS scan without overlay.
+* Microsoft Azure connector has been updated. Previously with UForge the machine image was publish as a "vhd" blob file in the Azure cloud Account. Now an image will be accessible in the cloud console from this blob file. In order to support this additional information must be entered in ``Credentials`` for Azure ARM connector.
```
Comment (jbremond) [inline admin/en/source/pages/changelog.rst]: Microsoft Azure ARM connector

File: end-user/en/source/pages/credentials/user-cloud-accounts.rst [hunk 8]
```diff
@@ -7,13 +7,37 @@ Managing Cloud Accounts
 
 In order to publish an image to a cloud using UForge AppCenter, you will need to add your cloud credentials to UForge. You must have a cloud account prior to setting up your credentials on the platform. Have all the information for your cloud account on hand before starting.
 
+.. note:: If you are an Amazon IAM user you will need to generate an X.509 Certificate following `these instructions <http://www.dowdandassociates.com/blog/content/howto-generate-an-x-dot-509-certificate-for-an-amazon-iam-user/>`_ prior to creating your Microsoft Azure classic cloud account on UForge.
```
Comment (jbremond) [inline end-user/en/source/pages/credentials/user-cloud-accounts.rst]: I don't understand this note

File: admin/en/source/pages/changelog.rst (hunk 7 above)
Comment (emuus) [inline admin/en/source/pages/changelog.rst]: ok

File: admin/en/source/pages/changelog.rst (hunk 6 above)
Comment (emuus) [inline admin/en/source/pages/changelog.rst]: ok

File: end-user/en/source/pages/credentials/user-cloud-accounts.rst (hunk 8 above)
Comment (emuus) [inline end-user/en/source/pages/credentials/user-cloud-accounts.rst]: This note was added a while back following input from PS

**Pull request merged**